process/wsindexer/indexer.go

```diff
@@ -49,6 +53,20 @@ func (i *indexer) GetOperationsMap() map[data.OperationType]func(d []byte) error
 	}
 }
 
+func (i *indexer) ProcessPayload(payload *data.PayloadData) error {
```

Comment:
Please also return errors here and set `BlockingAckOnError` to false

Reply:
changed for the part where the function call returns an error